docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+#Sphinx documentation
+!requirements.txt
```

Comment:
No need for the comment. It doesn't add any value here.
```suggestion
!requirements.in
!requirements.txt
```

Reply:
okay, I fix